### PR TITLE
Clearer install instruction

### PR DESCRIPTION
**Pull request by pierreozoux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ This WordPress plugin allows you to **use Discourse as a community engine for yo
 
 ## Installation
 
+### Plugin manager from your `wp-admin`
+
+Download the [zip](https://github.com/discourse/wp-discourse/archive/master.zip) and upload it to your wordpress instance from `wp-admin`/`plugins`/`Add new`/`Upload Plugin`
```
Comment (retlehs) [inline README.md]: should recommend downloading from the github releases page and not the master zip file

Comment (retlehs) [inline README.md]: > `wp-admin`/`plugins`/`Add new`/`Upload Plugin` 

this looks weird. why not just say:

> Download the latest release and upload it in the WordPress admin from Plugins > Add New > Upload Plugin

**Comment by pierreozoux** [comment]
Agreed :)

**Comment by retlehs** [comment]
thanks! can you squash into one commit please?

**Comment by pierreozoux** [comment]
Yes, sorry I could have done it directly :/ and the rebase didn't work as expected :/ but I opened another PR.

**Pull request closed without merging**